pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -135,7 +135,7 @@ public static String jsonPathString(Object object, String jsonPath)
     if (jsonValue instanceof String) {
       return (String) jsonValue;
     }
-    return JsonUtils.objectToString(jsonValue);
```

Comment:
during the null work, we introduced the convention to annotate these functions Nullable, as well as the params (check out examples in ObjectFunctions). Perhaps good to add here too